### ISSUE TITLE
Expire not used in execute

### ISSUE THREAD

**Issue by raxan** [mikecao/sparrow#32]
In execute() method the param $expire seem not used, here:

https://github.com/mikecao/sparrow/blob/master/sparrow.php#L756-L762

seem that fetch() check inside cached file expiration, here for example:

https://github.com/mikecao/sparrow/blob/master/sparrow.php#L1297

is this an issue or missing check of expiration? seem only file cache check it.

thanks

**Comment by mikecao** [comment]
You're right, it looks like the `execute` method doesn't use it. However everything should still work. Only the file cache checks the expire value, because all the other caches, like `memcached` handle expiration natively. See the `store` method.
